app/controllers/check_ins_controller.rb

```diff
@@ -17,6 +17,7 @@ def create
 	end
 
   def confirmation
+    @checkin = CheckIn.includes('user').last
```

Comment:
@joshkennedy My ruby is a bit rusty...is this line correct? I can't recall where this would be used but I think it's in the app/views/check_ins/confirmation.html.erb, and I don't see a reference to `@checkin` there. I see above that `@check_in` is used, is that what's supposed to be here too?